dlrover/python/elastic_agent/diagnosis/diagnosis_agent.py

```diff
@@ -49,35 +48,44 @@
 )
 from dlrover.python.diagnosis.inferencechain.inferenceoperator.operator import (  # noqa: E501
     get_training_failure_operators,
-    get_worker_diagnosis_operators,
     get_worker_observe_operators,
+    get_worker_resolve_operators,
 )
 from dlrover.python.elastic_agent.context import get_agent_context
 from dlrover.python.elastic_agent.master_client import MasterClient
 
 
 class DiagnosisAgent(Singleton):
-    def __init__(
-        self,
-        training_log_file: str,
-        errors: str,
-    ):
+    def __init__(self):
         self._client = MasterClient.singleton_instance()
-        self._training_log_file = training_log_file
-        self._errors = errors
+        self._training_log_file = ""
+        self._errors = ""
         self._stopped = False
-        self._observe_problems: List[Inference] = [
-            Inference(
-                name=InferenceName.WORKER,
-                attribution=InferenceAttribute.COLLECT,
-                description=InferenceDescription.METRICS,
-            ),
-        ]
+        # The key is the time interval in seconds
+        self._observe_problems: Dict[int, List[Inference]] = {
```

Comment:
![image](https://github.com/user-attachments/assets/2c494a84-1650-47a1-b94a-a6e027cf10ef)
Should add a notice if so: 'the minimum interval second is 5'.